universal-login-commons/lib/core/constants/test.ts

```diff
@@ -45,7 +45,12 @@ export const TEST_TOKEN_DETAILS = [
   {
     address: TEST_TOKEN_ADDRESS,
     symbol: 'DAI',
-    name: 'MockToken',
+    name: 'MockDAIToken',
+  },
+  {
+    address: TEST_CONTRACT_ADDRESS,
```

Comment:
Let's introduce `TEST_TOKEN_ADDRESS_2` or `TEST_SAI_ADDRESS` because we use `TEST_CONTRACT_ADDRESS` as a wallet contract address. I think that we should also rename `TEST_CONTRACT_ADDRESS` to `TEST_WALLET_CONTRACT_ADDRESS` to avoid such issues in the future.